c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -268,6 +268,21 @@ typedef enum {
     LSM_DISK_TYPE_HYBRID = 54,
 } lsm_disk_type;
 
+typedef enum {
+    LSM_DISK_LINK_TYPE_UNKNOWN =  -1,
+    LSM_DISK_LINK_TYPE_FC = 0,
+    LSM_DISK_LINK_TYPE_SSA = 2,
+    LSM_DISK_LINK_TYPE_SBP = 3,
+    LSM_DISK_LINK_TYPE_SRP = 4,
+    LSM_DISK_LINK_TYPE_ISCSI = 5,
+    LSM_DISK_LINK_TYPE_SAS = 6,
+    LSM_DISK_LINK_TYPE_ADT = 7,
+    LSM_DISK_LINK_TYPE_ATA = 8,
+    LSM_DISK_LINK_TYPE_USB = 9,
+    LSM_DISK_LINK_TYPE_SOP = 0xa,
+    LSM_DISK_LINK_TYPE_PCIE = 0xb,
+} lsm_disk_link_type;
```

Comment:
Any specific reason to make the last two hex, instead or remaining in base 10 like the others?